app/retail/emails.py

```diff
@@ -518,10 +518,50 @@ def render_funder_stale(github_username, days=60, time_as_str='a couple months')
     return response_html, response_txt
 
 
-def render_new_bounty(to_email, bounties, old_bounties, offset=3, trending_quests=[]):
+def render_new_bounty(to_email, bounties, old_bounties, offset=3, quest_of_the_day={}, upcoming_grant={}, upcoming_hackathon={}, latest_activities={}, from_date=date.today(), days_ago=7):
     from townsquare.utils import is_email_townsquare_enabled, is_there_an_action_available
-    email_style = (int(timezone.now().strftime("%-j")) + offset) % 24
+    from dashboard.models import Profile
+    from inbox.models import Notification
     sub = get_or_save_email_subscriber(to_email, 'internal')
+    if settings.DEBUG and False:
+        # for debugging email styles
+        email_style = 26
+    else:
+        email_style = (int(timezone.now().strftime("%-j")) + offset) % 24
+
+    # Get notifications count from the Profile.User of to_email
+    try:
+        profile = Profile.objects.filter(email__iexact=to_email).last()
+    except Profile.DoesNotExist:
+        pass
```

Comment:
why do we want to pass here instead of failing?

Reply:
It seems all emails that come at `to_email` need not have a profile: https://github.com/gitcoinco/web/pull/6471#issuecomment-623663597 @owocki. So if we fail here the email won't be send. Correct? 
Refernece: https://github.com/gitcoinco/web/pull/6471#discussion_r421056147